torch/csrc/jit/ir.h

```diff
@@ -139,6 +142,24 @@ struct Scope {
     }
     return current;
   }
+  bool hasConstant(std::string name) {
+    return constants_.find(name) != constants_.end();
+  }
+  int getConstant(std::string name) {
```

Comment:
`const` (`noexcept`)

Reply:
Are you sure you want the 0 to be returned when the name is not already in the map?

Reply:
Otherwise use `map.at(name)` which also has a const version